01-path-basics/103-kubernetes-concepts/readme.adoc

```diff
@@ -339,6 +339,10 @@ Watch the status of the Pod:
 
 `OOMKilled` shows that the container was terminated because it ran out of memory.
 
+To correct this, we'll need to re-create the pod with higher memory limits.  
+
+Although it may be instinctive to simply adjust the memory limit in the existing pod definition and re-apply it, https://github.com/kubernetes/kubernetes/issues/39060#issuecomment-268449812[Kubernetes does not currently support changing resource limits on running pods], so we'll need to first delete the existing pod, then recreate it.
```

Comment:
Is it necessary to reference the issue? Can we instead highlight that Kubernetes does not support in-place adjustments to pod resources?

If this will be fixed, then I can see tracking the issue.  However, if it's not on the roadmap, then we should leave the issue out.

Reply:
I was just being verbose :) No need to include the link reference.

Reply:
Can you fix and send a new commit?